driver-core/src/main/com/mongodb/operation/AggregateOperation.java

```diff
@@ -265,6 +300,10 @@ private BsonDocument asCommandDocument(final ConnectionDescription description)
         if (allowDiskUse != null) {
             commandDocument.put("allowDiskUse", BsonBoolean.valueOf(allowDiskUse));
         }
+        if (!readConcern.isServerDefault()) {
+            commandDocument.putAll(readConcern.asDocument());
```

Comment:
Shouldn't this be:

    commandDocument.put("readConcern", readConcern.asDocument());

?